pandas/plotting/_matplotlib/core.py

```diff
@@ -667,10 +665,10 @@ def _plot(cls, ax, x, y, style=None, is_errorbar=False, **kwds):
             if style is not None:
                 args = (x, y, style)
             else:
-                args = (x, y)
+                args = (x, y)  # type:ignore
```

Comment:
if adding ignores can you add the mypy error code and the mypy error message as a comment.

In this case, it's a relative simple cleanup. add a variable type declaration for args?